draft-ietf-quic-qpack.md

```diff
@@ -871,21 +871,38 @@ the decoder.
 
 ### Blocked Eviction
```

Comment:
The title here might be better as "Blocked Dynamic Table Insertions".  Evictions don't get blocked, they are the immovable object here.  It's more the case that insertions are what get blocked.